### DISCUSSION TITLE
Far coesistere intteruttore e lampadina smart

### DISCUSSION THREAD

**Discussion by carist1** [indomus/forum#1877] [GENERALE - Discussioni generiche]
Buon giorno a tutti,
Possiedo due faretti smart che sono collegati ad un interruttore il problema è che molte volte mi ritrovo o la mia compagna o degli ospiti in casa a spegnere accidentalmente le luci dall'interruttore... Mi tocca poi riattivare il circuito per ripristinare il tutto... C'è un modo per far coesistere l'interruttore di casa con le lamapdine smart? Ho visto che ci sono dei "relè" domotici online ma non so se potrebbero servire nel mio caso... 

Per ulteriori informazioni:
Le lampadine smart sono tutte di marche differenti collegati all'app smart life e come assistente vocale uso Alexa.

**Comment by snakuzzo** [comment]
Shelly o Sonoff ad esempio

**Reply by nromagna** [reply]
Confermo consiglio di @snakuzzo, ma a questo punto NON servirebbero più le lampadine smart ma normalissime lampadine led.

**Reply by snakuzzo** [reply]
Beh...non è detto. Magari vuole cambiare i colori e/o intensità utilizzando le lampadine smart già in possesso

**Reply by nromagna** [reply]
Hmmm hai ragione, non ci avevo pensato al colore...

**Reply by carist1** [reply]
Ci avevo pensato pure io a levare le lampadine smart ma il problema sarebbe sempre lo stesso in quanto, a questo interruttore è stata creata una spina che va sul contro soffitto a questa vi è collegata una ciabatta in cui sono state collegate tramite presa normale (create da me) le due lampadine ed infine il modem in quanto in un altro posto non si poteva mettere per questo mi serve che se anche si tocca il pulsante per spegnere o accendere ci sia sempre corrente e che magari solo lampadine ricevano l'input di spegnere o accendersi lasciano in pace il modem.
Spero sia abbastanza chiaro

**Reply by nromagna** [reply]
Ma se il filo che alimenta tutto è unico mica puoi fare altro... a prescindere da quello che ci metti.
Quindi o togli completamente l'interruttore a muro, in modo che la ciabatta sia sepre alimentata, e poi dalla ciabatta ai faretti metti due sonoff (o a questo punto lasci le lampadine smart) oppure fai arrivare alla ciabatta un altro cavo (sempre alimentato per il modem) e ritorni al primo consiglio.
Oppure iscruisci la moglie a NON toccare l'interruttore.

**Reply by carist1** [reply]
Il problema non è solo la moglie ma gli ospiti e adesso vedo come ingegnarmi. Grazie per i consigli
